trollsched/drawing.py

```diff
@@ -163,10 +171,9 @@ def save_fig(pass_obj,
     if not os.path.exists(directory):
         logger.debug("Create plot dir " + directory)
         os.makedirs(directory)
-    filename = os.path.join(
-        directory,
-        (rise + '_' + pass_obj.satellite.name.replace(" ", "_") + '_' + pass_obj.instrument.replace("/", "-") + '_' + fall + extension))
-
+    filename = os.path.join(directory, (rise + '_' +
```

Comment:
You could use `'{rise}_{sat_name}...{extension}'.format(**components)` here to increase readability.

Reply:
Fixed, thx